### PR TITLE
Portfolio Mobile menu

### PR DESCRIPTION
**Pull request by asnganje**
-Applied javascript for hamburgers to display the mobile menu 
-Added close button for users to click and the menu disappears 
-connected the mobile menu options to the correct parts of the page

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -509,6 +509,42 @@ ul {
...
+  flex-direction: column;
+  position: absolute;
+  top: 72px;
+  left: 0;
+  float: none;
+  height: auto;
+  background: #3c3a39;
+  width: 100%;
+  align-items: center;
+}
```
Comment (vechicin) [inline styles.css]: The style doesn't quite match that of the template design.

Screen Shot 2022-07-18 at 16 29 47

Screen Shot 2022-07-18 at 16 30 32

Make sure the menu covers the entirety of the screen in both axis and make sure the style of the menu matches that of the template design once you enlarge it.

Screen Shot 2022-07-18 at 16 31 56

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @asnganje ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Good job implementing the mobile menu functionalities

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mugishasam123 — CHANGES_REQUESTED** [review]
Hi @asnganje  👋 , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
alt


## Suggested changes
_Check the comments under the review._

- [ ] Please you should make expanded mobile menu fixed at top even on scrolling to improve user experience.so kindly Ipmlement that feature to make you project professional .when you scroll also the expanded navbar is also being scrolled which is not a good practice, you got this 👍 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by mirouhml — APPROVED** [review]
# APPROVED 🟢

Hi @asnganje, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] I suggest that you give your menu some `padding-top` so that the `x button` isn't so close to the edge, it would make it look better in my opinion.

image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**